les/serverpool.go

```diff
@@ -96,6 +96,7 @@ var (
 	sfQueried          = serverPoolSetup.NewFlag("queried")
```

Comment:
Now the `sfQuery` seems not a good name, let's rename it to `sfCanQuery`

Reply:
When this flag is set the query is always and instantly started so while the flag is active it does not represent a state where a query might be started, rather one where it has been started. So I'd leave this one as is.